packages/ERTP/src/amountMath.js

```diff
@@ -148,9 +143,9 @@ const optionalBrandCheck = (allegedBrand, brand) => {
 };
 
 /**
- * @template {AmountValue} [V=AmountValue]
- * @param {Amount<V>} leftAmount
- * @param {Amount<V>} rightAmount
+ * @template {AssetKind} [K=AssetKind]
+ * @param {Amount<K>} leftAmount
+ * @param {Amount<K>} rightAmount
  * @param {Brand | undefined} brand
```

Comment:
```suggestion
 * @param {Brand<K> | undefined} brand
```

Pinning down things like the kind is the reason to supply the optional brand.